bot/resources/tags/type-hint.md

```diff
@@ -0,0 +1,16 @@
+**Type Hints**
+
+A typehint indicates what type something should be. For example,
```

Comment:
```suggestion
A typehint indicates what type something is expected to be. For example,
```

Would this be an improvement in wording?

Reply:
"Type hint(s)" should be two separate words

Reply:
yes